pvlib/modelchain.py

```diff
@@ -161,9 +161,9 @@ class ModelChainResult:
 
     # per DC array information
     total_irrad: Optional[PerArray[pd.DataFrame]] = field(default=None)
-    """ DataFrame (or tuple of DataFrame, one for each array) containing
-    columns ``'poa_global'``, ``'poa_direct'`` ``'poa_diffuse'``,
-    ``poa_sky_diffuse'``, ``'poa_ground_diffuse'`` (W/m2); see
+    """ DataFrame (or tuple of DataFrame, one for each array containing
```

Comment:
```suggestion
    """ DataFrame (or tuple of DataFrame, one for each array) containing
```
This parenthesis should be retained